src/cdkutils/test_helpers/cloudwatch_alarm_simulator.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any, Dict, List
+
+from aws_cdk import aws_cloudwatch
+
+ALARM_STATE_OK = "OK"
+ALARM_STATE_ALARM = "ALARM"
+
+BREACHING_RULES = [
+    lambda defn, metric: defn.treat_missing_data == aws_cloudwatch.TreatMissingData.BREACHING and metric is None,
+    lambda defn, metric: defn.comparison_operator == aws_cloudwatch.ComparisonOperator.LESS_THAN_THRESHOLD
+    and metric < defn.threshold,
+    lambda defn, metric: defn.comparison_operator == aws_cloudwatch.ComparisonOperator.LESS_THAN_OR_EQUAL_TO_THRESHOLD
+    and metric <= defn.threshold,
+    lambda defn, metric: defn.comparison_operator
+    == aws_cloudwatch.ComparisonOperator.GREATER_THAN_OR_EQUAL_TO_THRESHOLD
+    and metric >= defn.threshold,
+    lambda defn, metric: defn.comparison_operator == aws_cloudwatch.ComparisonOperator.GREATER_THAN_THRESHOLD
+    and metric > defn.threshold,
+]
+
+
+class MetricException(Exception):
+    pass
+
+
+class CloudWatchAlarmSimulator:
+    def __init__(self, alarm_definition: aws_cloudwatch.AlarmProps):
```

Comment:
[LOW]

Return type on init missing

```suggestion
    def __init__(self, alarm_definition: aws_cloudwatch.AlarmProps) -> None:
```